### PR TITLE
Add spritz job status badges

### PR DESCRIPTION
**Pull request by MechaRyRy**
## Problem
We do not know immediately when opening the Spritz landing page what the current release is and whether the previous build was successful.

## Solution
Add badges to the readme and remove the unnecessary licence. The licence is provided as a badge and is included in the sources.

### Screenshots
screen shot 2017-12-05 at 14 03 18

### Paired with
Nobody.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+[![Build status](https://ci.novoda.com/buildStatus/icon?job=spritz)](https://ci.novoda.com/job/spritz/lastBuild/console) [![Download](https://api.bintray.com/packages/novoda/maven/spritz/images/download.svg)](https://bintray.com/novoda/maven/spritz/_latestVersion) [![](https://raw.githubusercontent.com/novoda/novoda/master/assets/btn_apache_lisence.png)](LICENSE.txt)
```
Comment (rock3r) [inline README.md]: `btn_apache_lisence` lol we should likely fix this — but more importantly ❌  it should link to `https://github.com/novoda/spritz/blob/master/LICENSE` (notice, no `.txt` extension)

**Review by rock3r — CHANGES_REQUESTED** [review]
I wouldn't remove the licence statement from the readme, but rather replace it with a link to the `LICENSE` file. You need to fix the link in the licence badge too

File: README.md (hunk 1 above)
Comment (frapontillo) [inline README.md]: You could even replace the image using the one from shields.io:

[license]()

```
[![license](https://img.shields.io/github/license/novoda/spritz.svg)]()
```

**Review by frapontillo — CHANGES_REQUESTED** [review]

**Comment by MechaRyRy** [comment]
> I wouldn't remove the licence statement from the readme

@rock3r Can we avoid adding it back now that I link directly to our license in the badge?

**Review by frapontillo — APPROVED** [review]

**Review by tasomaniac — APPROVED** [review]

**Review by rock3r — APPROVED** [review]

**Pull request merged**